open_academy/models/session.py

```diff
@@ -20,3 +20,20 @@ def _compute_taken_seats(self):
             if record.number_of_seats > 0:
                 taken_seats = round((len(record.attendees_ids) * 100) / record.number_of_seats)
             record.taken_seats = taken_seats
+
+    @api.onchange('number_of_seats','attendees_ids')
+    def _onchange_invalid_values(self):
+        if self.number_of_seats < 0:
+            return {
+                'warning': {
+                'title': "Something bad happened.",
+                'message': "Number of seats must be greater than 0."
```

Comment:
@sebasdrk17 maybe is not part of the subject but shouldn't be nice to add translations? I believe is useful that they work with this already.

Reply:
Yes, there is a section for that further in the course

I think we can leave this PR without translations, but I do think that we should start translating, so the next PR will have translations, what do you think?

Reply:
Seems ok, but I don't want to impose my rules :s just asked, whatever you consider the best :+1: